src/omnicore/omnicore.cpp

```diff
@@ -1812,6 +1812,7 @@ static int msc_initial_scan(int nFirstBlock)
         }
 
         unsigned int nTxNum = 0;
+        uint32_t nBlockFound = 0;
```

Comment:
Imho the name is a bit confusing and sounds as if it represents a number of blocks. Maybe `nTxsInBlock` or  `nOmniTxs`? Was there a particular reason to use `uint32_t` instead of `unsigned int`? `unsigned int` is the matching type of `mastercore_handler_block_end`, so it seems more appropriate.